src/components/Popups/ReplyField/PopupList.scss

```diff
@@ -0,0 +1,22 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.ba-PopupList {
+    font-size: 13px;
+
+    .ba-Popup-arrow {
+        visibility: hidden;
+    }
+
+    .ba-Popup-content {
+        padding-top: 10px;
+        background-color: $white;
+        border: solid 1px $bdl-gray-30;
+        border-radius: 4px;
+        box-shadow: 0 1px 2px 0 rgba(0, 0, 0, .05);
+    }
+
+    .ba-PopupList-startState {
+        padding: 10px;
```

Comment:
Should this padding apply to the whole `PopupList` (startState as well as eventual  at-mention content) or is 
 it specific to the start state?